src/App.js

```diff
@@ -41,18 +46,18 @@ function App() {
   return (
     <Container>
       <PwaPrompt />
-      <header>
-        <PageSelection
-          pages={App.pages}
-          allProducts={allProducts}
-          createProduct={createProduct}
-          removeProduct={removeProduct}
-          updateProduct={updateProduct}
-          increaseLikeCount={increaseLikeCount}
-          setCurrentProduct={setCurrentProduct}
-          currentProduct={currentProduct}
-        />
-      </header>
+      <StaticHeader staticHeader={staticHeader} />
```

Comment:
It will be more clear if you break `staticHeader` object when passing as a prop. This way is easier to
understand what the props are changing in the component. Also, because `staticHeader` has the same name as the component, it made a little more confusing:
```suggestion
      <StaticHeader title={staticHeader.title} onBackButtonClick={staticHeader.onBackButtonClick} />
```

Reply:
yeah. I was hesitant about this point, thanks 
c1b2d79